mongodbatlas/resource_mongodbatlas_cluster_outage_simulation.go

```diff
@@ -0,0 +1,193 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mwielbut/pointy"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	errorClusterOutageSimulationCreate  = "error starting MongoDB Atlas Cluster Outage Simulation for Project (%s), Cluster (%s): %s"
+	errorClusterOutageSimulationRead    = "error getting MongoDB Atlas Cluster Outage Simulation for Project (%s), Cluster (%s): %s"
+	errorClusterOutageSimulationDelete  = "error ending MongoDB Atlas Cluster Outage Simulation for Project (%s), Cluster (%s): %s"
+	errorClusterOutageSimulationSetting = "error setting `%s` for MongoDB Atlas Cluster Outage Simulation: %s"
+)
+
+func resourceMongoDBAtlasClusterOutageSimulation() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceMongoDBClusterOutageSimulationCreate,
+		ReadContext:   resourceMongoDBAClusterOutageSimulationRead,
+		UpdateContext: resourceMongoDBClusterOutageSimulationUpdate,
+		DeleteContext: resourceMongoDBAtlasClusterOutageSimulationDelete,
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"cluster_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
```

Comment:
looking at the open [api doc ](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Cluster-Outage-Simulation/operation/getOutageSimulation), there is an ID field that we are not tracking. Is this expected?

Reply:
The `id` field is being mapped to `simulation_id`